template/semantic-ui/layout/admin.html

```diff
@@ -1,13 +1,7 @@
 
 <div class="atk-layout">
```

Comment:
Can an html ID be added here?

Reply:
No. This is how it is defined for this layout.

Reply:
Strange, the top-most element can not have an html id?

Reply:
Related with https://github.com/atk4/ui/issues/1028#issuecomment-606783135

Reply:
This pr is not about adding ID to the layout. Perhaps another time where we could add it to all layout as well.